arm9/source/main.c

```diff
@@ -49,8 +49,14 @@ static int load_file(const char *filename, uint32_t addr)
 	return 1;
 }
 
+static bool isKtr(void)
```

Comment:
I think in the current code all the function and variable names are in `snake_case`.
We should also agree on the coding style for this repo and arm9linuxfw :)

Reply:
the original arm9linuxfw was written in C++, that's why the port still uses camelCase but that should be easy to fix